examples/spark_udf/spark_udf_with_prebuilt_env.py

```diff
@@ -0,0 +1,42 @@
+"""
+This example code shows how to use `mlflow.pyfunc.spark_udf` with Databricks Connect
+outside Databricks runtime.
+"""
+
+import os
+
+from databricks.connect import DatabricksSession
+from sklearn import datasets
+from sklearn.neighbors import KNeighborsClassifier
+
+import mlflow
+
+spark = DatabricksSession.builder.remote(
+    host=os.environ["DATABRICKS_HOST"],
+    token=os.environ["DATABRICKS_TOKEN"],
+    cluster_id="<cluster id>",  # get cluster id by spark.conf.get("spark.databricks.clusterUsageTags.clusterId")
+).getOrCreate()
+
+X, y = datasets.load_iris(as_frame=True, return_X_y=True)
+model = KNeighborsClassifier()
+model.fit(X, y)
+
+with mlflow.start_run():
+    model_info = mlflow.sklearn.log_model(model, "model")
+
+model_uri = model_info.model_uri
+
+# The prebuilt model environment archive file path.
+# To build the model environment, run `mlflow.pyfunc.build_model_env` in Databricks runtime,
+# and save the built environment archive file to a UC volume path
+# and then download the built env archive file to local disk from the saved UC volume path.
```

Comment:
does databricks SDK support downloading a file in a UC volume?

Reply:
Yes. We have API for it.`WorkspaceClient().files.download(<path>) `

Reply:
can we add it in the script?

Reply:
added.

Reply:
I meant add the code, not comments

Reply:
updated.